src/components/Tabs.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Tabs.scss';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+  selectedTabContent,
+}) => (
+  <div className="tabs">
+    <ul className="tabs__list">
+      {tabs.map(tab => (
+        <li key={tab.id} className="tabs__item">
+          <button
+            type='button'
+            className={
+              `tabs__btn ${tab.id !== selectedTabId
+                ? 'tabs__btn--active'
+                : ''}`
+            }
```

Comment:
```suggestion
            className={
              `tabs__btn ${tab.id !== selectedTabId
                ? 'tabs__btn--active'
                : ''}`
            }
```
Можно использовать Classnames.

Reply:
Не можно, а нужно 😄